packages/babel-plugin-transform-modules-commonjs/src/index.js

```diff
@@ -10,9 +10,16 @@ import {
 import simplifyAccess from "@babel/helper-simple-access";
 import { template, types as t } from "@babel/core";
 
+import babelPluginDynamicImportNode from "babel-plugin-dynamic-import-node";
+
 export default declare((api, options) => {
   api.assertVersion(7);
 
+  // TODO: expose a better interface
+  const transformImportCall = Function.call.bind(
+    babelPluginDynamicImportNode(api).visitor.Import,
```

Comment:
Please ignore the ugliness of this line. If this PR gets approved, I will open a PR in the `babel-plugin-dynamic-import-node` repo to directly export the `transformImportCall` function.